hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -1592,4 +1595,43 @@ public OpenKeySession createFile(OmKeyArgs args,
     }
     return statusList;
   }
+
+  @Override
+  public List<RepeatedOmKeyInfo> listTrash(String volumeName,
+      String bucketName, String startKeyName, String keyPrefix, int maxKeys)
+      throws IOException {
+
+    Preconditions.checkArgument(Strings.isNullOrEmpty(volumeName),
+        "The volume name cannot be null or " +
+        "empty.  Please enter a valid volume name or use '*' as a wild card");
+
+    Preconditions.checkArgument(Strings.isNullOrEmpty(bucketName),
+        "The bucket name cannot be null or " +
+        "empty.  Please enter a valid bucket name or use '*' as a wild card");
```

Comment:
As a future JIRA, I was thinking it might be useful to get just the list of buckets that are deleted. It might be interesting.

Reply:
I just created HDDS-2457 to track that.